packages/core/adder/config.ts

```diff
@@ -25,18 +20,22 @@ export type Scripts<Args extends OptionDefinition> = {
 	condition?: ConditionDefinition<Args>;
 };
 
+export type SvApi = {
+	file: (path: string, content: (content: string) => string) => string;
```

Comment:
perhaps this should return `void` instead of a `string`? not sure of a use-case where returning a `string` would help
```suggestion
	file: (path: string, edit: (content: string) => string) => void;
```